Jellyfin.Api/Controllers/PackageController.cs

```diff
@@ -127,7 +127,6 @@ public async Task<IEnumerable<PackageInfo>> GetPackages()
         /// <response code="200">Package repositories returned.</response>
         /// <returns>An <see cref="OkResult"/> containing the list of package repositories.</returns>
         [HttpGet("Repositories")]
-        [Authorize(Policy = Policies.DefaultAuthorization)]
```

Comment:
That seems like a mistake?

Reply:
It's applied at the controller level as described here: https://github.com/jellyfin/jellyfin/pull/3812#discussion_r466034565